XML/TestCases/FunctionalTests-TVM.xml

```diff
@@ -14,4 +14,23 @@
       <VMGeneration>2</VMGeneration>
     </SetupConfig>
   </test>
+  <test>
+    <testName>TVM-TEST-COMPATIBILITY</testName>
+    <testScript>TVM-TestCompatibility.sh</testScript>
+    <files>.\Testscripts\Linux\TVM-TestCompatibility.sh,.\Testscripts\Linux\utils.sh,.\Tools\tvminfo</files>
```

Comment:
Is it a tool that can be downloaded somewhere?

Reply:
TestHelpers.psm1 and .gitignore file should be updated accordingly after it has been published.

Reply:
Removed the tool from here. The test script is not downloading the tool from a public repo.